kuma/search/tests/test_serializers.py

```diff
@@ -59,6 +59,25 @@ def test_document_serializer(self):
         assert isinstance(dict_data, dict)
         assert dict_data['id'] == result[0].id
 
+    def test_search_query_serializer(self):
+        search_serializer = SearchQuerySerializer(
+            data={'q': 'test'}
+        )
+        self.assertTrue(search_serializer.is_valid())
```

Comment:
I know it's a bit confusing but generally we use `assert` for tests. As much as possible. 
The only time we use `self.assertTrue` or `self.assertEqual` is if you're editing a test that already is a class based one. E.g. 
```python
class TestStuff(KumaTestCase):
    def test_life(self):
        self.assertTrue(meaning(), 42)
```

And if it's pure functions, we always use `assert`. I.e. 

```python
def test_life():
    assert meaning() == 42
```

In [this particular file](https://github.com/mdn/kuma/blob/b28612fe37dfba50bedf7d5e8e86c0a2b196c267/kuma/search/tests/test_serializers.py) there are class based tests. **But!** it's using `assert` already so stick to that for consistency. So, change all your `self.assert*` to `assert`. E.g.
```python
assert search_serializer.is_valid()
# or 
assert not search_serializer.is_valid()
# or
assert search_serializer.errors.keys() == ['q']
```